src/remote_gym/remote_environment.py

```diff
@@ -9,6 +10,7 @@
 from dm_env_rpc.v1 import dm_env_adaptor
 from gymnasium import Env
 from gymnasium.spaces import Box, Discrete, MultiDiscrete, Space
+from omegaconf import OmegaConf
```

Comment:
Not happy with omegaconf here either, Remote_args is only a Omega dict because its directly passed from the environment space. Thus, the whole omegaconf is a "hidden" dependency caused by another repo.